python_modules/libraries/dagster-k8s/dagster_k8s/client.py

```diff
@@ -731,6 +731,50 @@ def _has_container_logs(self, container_status):
 
         return False
 
+    def _get_job_status_str(self, job):
+        if not job.status:
+            return "Could not determine job status."
+
+        job_status = (
+            "Job status:"
+            + f"\n - starttime: {job.status.start_time.isoformat()}"
```

Comment:
nit: `start_time` instead of `starttime`

Reply:
Changed